src/ripple/basics/impl/PerfLogImp.cpp

```diff
@@ -273,6 +260,7 @@ PerfLogImp::run()
     {
         {
             std::unique_lock<std::mutex> lock(mutex_);
+            cond_.wait_until(lock, lastLog_ + setup_.logInterval);
```

Comment:
Except for the first iteration, this looks like the same logic to me. I don't object to the change, but I'm not sure I see the reason for the change.

Reply:
It doesn't report to the old file one more time after a rotate command has been received, and it shuts down immediately instead of trying to make one last report. It feels "more right" to me.

Reply:
That makes sense. Thanks for the clarification.